packages/image_picker/image_picker_windows/README.md

```diff
@@ -0,0 +1,16 @@
+# image\_picker\_windows
+
+A Windows implementation of [`image_picker`][1].
+
```

Comment:
This should mention that image source is not currently supported.

Reply:
Done.